python/rpdk/java/templates/init/guided_aws/StubListHandler.java

```diff
@@ -0,0 +1,31 @@
+package {{ package_name }};
+
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.OperationStatus;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class {{ operation }}Handler extends BaseHandler<CallbackContext> {
+
+    @Override
+    public ProgressEvent<{{ pojo_name }}, CallbackContext> handleRequest(
+        final AmazonWebServicesClientProxy proxy,
+        final ResourceHandlerRequest<{{ pojo_name }}> request,
+        final CallbackContext callbackContext,
+        final Logger logger) {
+
+        final List<ResourceModel> models = new ArrayList<>();
+
+        // TODO : put your code here
```

Comment:
Should we do a bit more here to demonstrate a sensible call pattern with `NextToken` handling for example?

Reply:
sometimes in list handler pattern wont work as callback traces the response values, which might exceed the size limit